pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md

```diff
@@ -29,305 +29,241 @@ You can activate the autoscaler on several node pools, each of which can have a
 
-Now we can patch the *Prime Numbers* deployement to augment the replicas to 12, that should be enough to activate the scaling up of the node pool.
+Now we can set the `relicas` number to 12, that should be enough to activate the scaling up of the node pool.
 
 ```bash
-kubectl patch deployment prime-numbers --type="merge" --patch='{"spec": {"replicas": 12}}'
+kubectl scale --replicas=12 deploy/python-cpu-load -n cluster-autoscaler
 ```
 
 ```console
-$ kubectl patch deployment prime-numbers --type="merge" --patch='{"spec": {"replicas": 12}}'
-deployment.apps/prime-numbers patched
-
-$ kubectl get pods
-NAME                             READY   STATUS    RESTARTS   AGE
-prime-numbers-5ffd8d7b84-2gmgh   0/1     Pending   0          22s
-prime-numbers-5ffd8d7b84-6spmf   1/1     Running   0          12m
-prime-numbers-5ffd8d7b84-87xc6   1/1     Running   0          4m57s
-prime-numbers-5ffd8d7b84-bv2sx   1/1     Running   0          4m57s
-prime-numbers-5ffd8d7b84-gmxjc   1/1     Running   0          4m57s
-prime-numbers-5ffd8d7b84-jm8mc   0/1     Pending   0          22s
-prime-numbers-5ffd8d7b84-jss9r   1/1     Running   0          4m57s
-prime-numbers-5ffd8d7b84-pw4f9   1/1     Running   0          22s
-prime-numbers-5ffd8d7b84-sb9vz   1/1     Running   0          4m57s
-prime-numbers-5ffd8d7b84-tltb8   1/1     Running   0          12m
-prime-numbers-5ffd8d7b84-vbzjw   1/1     Running   0          12m
-prime-numbers-5ffd8d7b84-wj9cf   0/1     Pending   0          22s
+$ kubectl scale --replicas=12 deploy/python-cpu-load -n cluster-autoscaler
+deployment.apps/python-cpu-load scaled
+
+$ kubectl get pods -n cluster-autoscaler
+NAME                               READY   STATUS    RESTARTS   AGE
+python-cpu-load-59bfff47f4-9lgnq   1/1     Running   0          3m54s
+python-cpu-load-59bfff47f4-c9ttt   1/1     Running   0          15s
+python-cpu-load-59bfff47f4-d6trc   1/1     Running   0          3m52s
+python-cpu-load-59bfff47f4-m4vcw   0/1     Pending   0          15s
+python-cpu-load-59bfff47f4-md69d   0/1     Pending   0          15s
+python-cpu-load-59bfff47f4-n8fgl   1/1     Running   0          15s
+python-cpu-load-59bfff47f4-nbwqd   1/1     Running   0          15s
+python-cpu-load-59bfff47f4-nbxcj   0/1     Pending   0          15s
+python-cpu-load-59bfff47f4-rbstz   0/1     Pending   0          15s
+python-cpu-load-59bfff47f4-tczf2   1/1     Running   0          15s
+python-cpu-load-59bfff47f4-vzq2g   1/1     Running   0          3m50s
+python-cpu-load-59bfff47f4-xqd72   1/1     Running   0          15s
 ```
 
 As you can see then with `kubectl get nodepools`, the autoscaler detects capacity has been reached and asks for a new node:
 
 ```console
 $ kubectl get nodepools
-NAME            FLAVOR   AUTO SCALED   [...]   ANTI AFFINITY   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   MIN   MAX
-nodepool-b2-7   b2-7     true          [...]   false           2         1         1            1           0     3
+
+kubectl get nodepools
```

Comment:
the command is displayed two times? :)

Reply:
fix https://github.com/ovh/docs/pull/5940/commits/bcc1b4a09339b41ea5fb0aadddc70dc9d6d9c97d